token/program-2022/src/extension/mod.rs

```diff
@@ -401,6 +480,23 @@ impl ExtensionType {
             ExtensionType::MintPaddingTest => pod_get_packed_len::<MintPaddingTest>(),
         }
     }
+
+    /// Get the associated account type
+    pub fn get_associated_account_type(&self) -> AccountType {
```

Comment:
wdyt about dropping "associated"?  When I see "associated_account" in a token context, my brain jumps to ATA immediately:
```suggestion
    pub fn get_account_type(&self) -> AccountType {
```

Reply:
Hehe I had the same thought, and then opted for consistency with `get_associated_type_len()`.  Happy to remove it